### PR TITLE
feat(native-stack): add support for large title attributes

### PR DESCRIPTION
**Pull request by davrosull**
This PR surfaces the `largeTitleFontFamily` and `largeTitleFontSize` now available in react-native-screens 2.0.0-alpha.5

### CODE REVIEW

File: packages/native-stack/package.json [hunk 1]
```diff
@@ -34,14 +34,14 @@
   "devDependencies": {
     "@react-native-community/bob": "^0.7.0",
     "del-cli": "^2.0.0",
-    "react-native-screens": "^2.0.0-alpha.3",
+    "react-native-screens": "2.0.0-alpha.5",
```
Comment (satya164) [inline packages/native-stack/package.json]:
```suggestion
    "react-native-screens": "^2.0.0-alpha.5",
```

File: packages/native-stack/package.json [hunk 2]
```diff
@@ -34,14 +34,14 @@
...
     "typescript": "^3.5.3"
   },
   "peerDependencies": {
     "@react-navigation/core": "^5.0.0-alpha.0",
     "react": "*",
     "react-native": "*",
-    "react-native-screens": "^2.0.0-alpha.3"
+    "react-native-screens": "2.0.0-alpha.5"
```
Comment (satya164) [inline packages/native-stack/package.json]:
```suggestion
    "react-native-screens": "^2.0.0-alpha.5"
```

**Review by satya164 — CHANGES_REQUESTED** [review]
Thanks for the PR. Can you update it to use `^` for versions?

**Comment by davrosull** [comment]
Updated with the suggested changes - thanks @satya164

**Review by satya164 — APPROVED** [review]

**Pull request merged**